Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed an issue where a composite binding would not be consecutively triggered after ResetDevice() has been called from the associated action handler [ISXB-746](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-746).
 - Fixed resource designation for "d_InputControl" icon to address CI failure.
 - Fixed an issue where a composite binding would not be consecutively triggered after disabling actions while there are action modifiers in progress [ISXB-505](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-505).
+- Fixed Missing default control scheme used by PlayerInput component are now correctly shown in the inspector [ISXB-818](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-818)
```

Comment:
Am I correct to assume the behavior with a missing control scheme is equivalent to having it set to Any (ignoring Inspector representation)?

Reply:
Seems like this is mainly targeting representation of control scheme when opening a project/asset with a previously configured control scheme that no longer exist, or e.g. deleting a control scheme.